src/tracing/options.ts

```diff
@@ -55,18 +55,32 @@ export type CaptureHttpUriParameters = (
 ) => void;
 
 export interface Options {
+  accessToken: string;
   endpoint?: string;
   serviceName: string;
-  accessToken: string;
-  serverTimingEnabled: boolean;
+  // Tracing-specific configuration options:
+  captureHttpRequestUriParams: string[] | CaptureHttpUriParameters;
   instrumentations: InstrumentationOption[];
-  tracerConfig: NodeTracerConfig;
+  propagatorFactory: PropagatorFactory;
+  serverTimingEnabled: boolean;
   spanExporterFactory: SpanExporterFactory;
   spanProcessorFactory: SpanProcessorFactory;
-  propagatorFactory: PropagatorFactory;
-  captureHttpRequestUriParams: string[] | CaptureHttpUriParameters;
+  tracerConfig: NodeTracerConfig;
 }
 
+export const allowedTracingOptions = [
```

Comment:
you could extract fields specific to tracing to a new type `TracingOptions`, and then this array can be of type `(keyof TracingOptions)[]`, but it's not a big deal

Reply:
Yes, the current type works just as well in compile-time, but  I need it in runtime. Do you know a way to compile runtime vars from types?